src/OrchardCore.Modules/OrchardCore.AdminDashboard/Migrations.cs

```diff
@@ -59,6 +59,11 @@ public int UpdateFrom2()
             );
 
             return 3;
-        }        
+        }
+
+        public void Uninstall()
+        {
+            SchemaBuilder.DropMapIndexTable<DashboardPartIndex>();
```

Comment:
It would also need to drop the indexes on these tables, or it will fail on SQL Server / MySql and Postgres

I'm not sure this is the best way forward with this.
I think we should discuss at triage

I personally think this is a good approach. Would require some small hooks in YesSql SchemaBuilder perhaps.

https://github.com/OrchardCMS/OrchardCore/issues/1212#issuecomment-515183361

Can we check also what happens when a module is disabled? Does it hit these methods?
That might be problematic if you are just disabling a module temporarily. 
(But I'm not sure if the DataMigration manager has anything implemented for `Uninstall`
keep a track of what was created in a document, and the commands required to remove it.

Reply:
Why don't we use the SchemaBuilder to do it all?